examples/.docksal/docksal.env

```diff
@@ -22,9 +22,15 @@ DOCKSAL_STACK=default
 # MySQL settings.
 # MySQL will be exposed on a random port. Use "fin ps" to check the port.
 # To have a static MySQL port assigned, copy the line below into the .docksal/docksal-local.env file
-# and replace the host port "0" with a unique host port number (e.g. MYSQL_PORT_MAPPING='33061:3306')
+# and replace the host port "0" with a unique host port number (e.g., MYSQL_PORT_MAPPING='33061:3306')
 MYSQL_PORT_MAPPING='0:3306'
 
+# Elastic Search settings.
```

Comment:
Since Elasticsearch is not part of the default stack, this does not belong here. Please remove.
Mention the "unique host port number" detail in the docs instead.

Reply:
Updated.